### DISCUSSION TITLE
Is this the same as tablericons.com?

### DISCUSSION THREAD

**Discussion by impressivewebs** [tabler/tabler-icons#128] [General]
Your project seems to be oddly similar to this one:

https://tablericons.com/

Both have 1250+ icons, same name, same icons, same search feature, very similar filter options...

Who copied who here?

**Comment by codecalm** [comment]
tabler-icons.io are official website od Tabler Icons :) page that you mentioned is not connected to our project :)

**Comment by impressivewebs** [comment]
@codecalm But you have the same exact icons. How is that possible if it's not connected in any way?

**Comment by codecalm** [comment]
Tabler Icons are under MIT license, so every person can redistribute icons legally

**Reply by GreatestFool** [reply]
Perhaps a bit late, consider the date today, but just to check I'm not completely drunk. while Tabler Icons are MIT, tabler-icons.io is the official one you made?

Sorry if it's a bit of a basic question. I'm tired and thought it best to ask for clarification.

**Reply by codecalm** [reply]
@GreatestFool That person created a page that bears a false resemblance to the official Tabler Icons website: [https://tabler-icons.io](https://t.co/YglBv68h0d) and generates false reach and misleads users through it. Moreover, it is outdated and falsely implies that the author of the icons is Csaba Kissi, not me.

**Comment by impressivewebs** [comment]
@codecalm Oh I see... It has your name at the bottom of that website. I didn't notice that before. Ok, that explains it, thanks.
